wlog-zap/internal/logger.go

```diff
@@ -147,8 +147,8 @@ func (l *zapLogger) SetLevel(level wlog.LogLevel) {
 
 func logOutput(logFn func(string, ...zap.Field), msg string, params []wlog.Param) {
 	entry := newZapLogEntry()
-	wlog.ApplyParams(entry, params)
-	logFn(msg, entry.Fields()...)
+	wlog.ApplyParams(entry, wlog.ParamsWithMessage(msg, params))
+	logFn("", entry.Fields()...)
```

Comment:
Can you add a comment that notes that "message" parameter is empty string because it has already been added to the parameters via the `ApplyParams` call if present?